src/components/ui/TextArea/TextArea.module.css

```diff
@@ -0,0 +1,70 @@
+.root {
+  width: 100%;
+  padding: 10px;
+  font-family: var(--font-montserrat);
+  font-size: 16px;
+  line-height: 1.2;
+  color: rgb(var(--rgb-foreground-secondary));
+  background-color: transparent;
+  border: 1px solid rgb(var(--rgb-foreground-secondary));
+  border-radius: var(--border-radius);
+  outline: none;
+  resize: vertical;
+  transition: var(--common-transition);
+}
```

Comment:
![123](https://github.com/user-attachments/assets/7c79a9fa-fce5-40f4-8a5f-c67045e16c54)
Would be nice to add min-height as well. At least 100 px but look through the design and define it

Reply:
@b3dyk Done!